src/scss/layout/_footer.scss

```
@@ -41,4 +41,63 @@
       }
     }
   }
+
+  @include viewport-desktop {
+    & {
```

And leave this properties out of the `{}`, like 
```suggestion
    @include viewport-desktop {
      align-items: center;
      display: flex;
      justify-content: space-between;
      margin: 0;
      padding: 0;
      
      &__data-links {
        .
        .
        .
      }
      .
      .
      .
    }
```